src/task.js

```diff
@@ -14,15 +14,16 @@ export class Duty {
 }
 
 export function removecompleted(arr) {
-  const todelete = [];
   arr.forEach((element) => {
     if (element.completed === true) {
-      todelete.push(element);
-      const filteredArray = arr.filter((item) => !todelete.includes(item));
-      window.localStorage.setItem('tasklist', JSON.stringify(filteredArray));
-      window.location.reload();
+      arr.splice(arr.indexOf(element), 1);
     }
   });
+  arr.forEach((element) => {
+    element.index = arr.indexOf(element);
+  });
+  window.localStorage.setItem('tasklist', JSON.stringify(arr));
+  window.location.reload();
```

Comment:
Good job working on the indexing.

Oops, you removed the `filter()` method from your code which is a hard requirement 🤦 , and also you introduce a new bug, `removecompleted` method does not remove all the completed tasks but just 1.

kindly fix these two issues. 🙏